FirebaseFunctions.podspec

```diff
@@ -1,72 +1,57 @@
-  s.name             = 'FirebaseFunctions'
-  s.version          = '8.12.0'
+  s.name                    = 'FirebaseFunctions'
+  s.version                 = '8.12.0'
   s.summary          = 'Cloud Functions for Firebase'
 
   s.description      = <<-DESC
 Cloud Functions for Firebase.
                        DESC
 
-  s.homepage         = 'https://developers.google.com/'
-  s.license          = { :type => 'Apache', :file => 'LICENSE' }
-  s.authors          = 'Google, Inc.'
-  s.source           = {
-    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+  s.homepage                = 'https://developers.google.com/'
+  s.license                 = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors                 = 'Google, Inc.'
+
+  s.source                  = {
+    :git => 'https://github.com/Firebase/firebase-ios-sdk.git',
     :tag => 'CocoaPods-' + s.version.to_s
   }
 
-  s.ios.deployment_target = '10.0'
-  s.osx.deployment_target = '10.12'
-  s.tvos.deployment_target = '10.0'
-  s.watchos.deployment_target = '6.0'
+  s.swift_version           = '5.3'
+
+  ios_deployment_target = '10.0'
+  osx_deployment_target = '10.12'
+  tvos_deployment_target = '10.0'
+  watchos_deployment_target = '6.0'
+
+  s.ios.deployment_target = ios_deployment_target
+  s.osx.deployment_target = osx_deployment_target
+  s.tvos.deployment_target = tvos_deployment_target
+  s.watchos.deployment_target = watchos_deployment_target
 
-  s.cocoapods_version = '>= 1.4.0'
-  s.prefix_header_file = false
+  s.cocoapods_version       = '>= 1.4.0'
+  s.prefix_header_file      = false
 
   s.source_files = [
-    'FirebaseFunctions/Sources/**/*',
-    'FirebaseAuth/Interop/*.h',
-    'FirebaseAppCheck/Interop/*.h',
-    'FirebaseCore/Internal/*.h',
-    'FirebaseMessaging/Interop/FIRMessagingInterop.h',
+    'FirebaseFunctions/Sources/**/*.swift',
   ]
-  s.public_header_files = 'FirebaseFunctions/Sources/Public/FirebaseFunctions/*.h'
 
-  s.dependency 'FirebaseCore', '~> 8.0'
+  s.dependency 'FirebaseCore', '~> 8.12'
+  s.dependency 'FirebaseCoreInternal', '~> 8.12'
+  s.dependency 'FirebaseAppCheckInterop', '~> 8.12'
+  s.dependency 'FirebaseAuthInterop', '~> 8.12'
+  s.dependency 'FirebaseMessagingInterop', '~> 8.12'
+  s.dependency 'FirebaseSharedSwift', '~> 8.12'
   s.dependency 'GTMSessionFetcher/Core', '~> 1.5'
 
-  s.pod_target_xcconfig = {
-    'GCC_C_LANGUAGE_STANDARD' => 'c99',
-    'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}"'
-  }
-
-  s.test_spec 'unit' do |unit_tests|
-    unit_tests.scheme = { :code_coverage => true }
-    unit_tests.source_files = [
-      'FirebaseFunctions/Tests/Unit/*.[mh]',
-      'FirebaseFunctions/Tests/SwiftUnit/**/*',
-      'SharedTestUtilities/FIRAuthInteropFake*',
-      'SharedTestUtilities/FIRMessagingInteropFake*',
-      'SharedTestUtilities/AppCheckFake/*.[mh]',
-  ]
-  end
+  # Maybe TODO: The unit tests depend on setting up a podspec for SharedTestUtilities
+  # OTOH, we are running them with SwiftPM.
```

Comment:
I think considering there's at least an integration test running on CocoaPods, I'd be comfortable only having unit tests on SwiftPM. Any build configuration issues would show up in the integration tests. WDYT?

Reply:
Agreed. Removed comment.